util/coverage/coverage.js

```diff
@@ -207,6 +207,11 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list contains contents that are not yet included in the latest cactbot release version.<br>So, you may see some contents are not covered by the release version although this list says cactbot covers that content.',
```

Comment:
```suggestion
    en: 'This list contains contents that are not yet included in the latest cactbot release version.<br>So, you may see some contents are not covered by the release version although this list says cactbot covers that content.',
  ja: 'このリストは最新バージョンに公開されていないコンテンツを含まれています。<br>そのため、リストに含まれているコンテンツは実際に公開されているリリースに存在はしないかもしれぬことをご理解してください。',
  cn: '该列表中可能存在未发布在cactbot最新发行版中的更新内容。<br>也就是说，即便某些副本支持在列表中已显示为支持状态，也可能暂未存在于您下载的版本中。',
```

Reply:
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date.',
```

Proposing a slight rewording here.  I think no need for `<br>` either, as it should wrap normally?

Reply:
I thought the content width balance would be better by adding line break.
And... I encountered a problem I didn't noticed. The emoji goes up when the sentence exceeds the width of the page.

![K-008](https://user-images.githubusercontent.com/4695371/122966824-be054800-d3c4-11eb-8af2-c13e04d89062.png)

Reply:
I think you could fix both of those (the really long line, and the emoji on the other line) with additions to `coverage.css`.